spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
```

Comment:
- [ ] It would be better if we implement the integration test for user signing in successfully and unsuccessfully. Kindly go back to the project guidelines and check if all requirements has been met

Reply:
@addod19 
Though the requirements request tests for the login page, this has not been a previously requested part of the wireframe build and has yet to be implemented at this stage. Please reference this [issue](https://github.com/microverseinc/curriculum-rails/issues/107) that was submitted to correct this requirement for this project